### ISSUE TITLE
Fix heading structure problem on component detail page

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#524]
Related to #459.

Because we're (I believe) reusing the code that displays a component title in multiple places, it can introduce a heading structure problem, since the component title is displayed at different conceptual levels. For example, in the screenshot below, we show the component title at the top of the content as an H1, which is semantically correct there, but then when we show the collection title as part of the collection context later, it is also displayed using an H1, even though in that context it follows an H3. So using the H1 in the second instance is semantically incorrect.

<img width="931" alt="Screen Shot 2019-07-19 at 3 23 24 PM" src="https://user-images.githubusercontent.com/101482/61569110-af74c000-aa3a-11e9-9889-1149d5b83cee.png">

---

Resolving this will probably require us to adjust how we're reusing the code used to display the component title but I'm going to label this ticket as design needed before making a specific recommendation because it could be affected by decisions we make when trying to [improve the overall layout of the component page](#519).

**Comment by ggeisler** [comment]
Superseded by #820 and related tickets.
